lib/charms/layer/jenkins/packages.py

```diff
@@ -130,20 +127,18 @@ def _setup_source(self, release):
         self._apt.add_source(source, key=key)
 
     def _bundle_download(self, path=None):
-        hookenv.log(
-            "Downloading bundle from %s" % self._jc.jenkins_repo)
+        hookenv.log("Downloading bundle from %s" % self._jc.jenkins_repo)
         self._jc.get_binary_package(path)
 
     def jenkins_upgradable(self):
         """
-            Verify if there's a new version of jenkins available.
-            Note: When bundle-site is not set the return will always
-            be True.
+        Verify if there's a new version of jenkins available.
+        Note: When bundle-site is not set the return will always
+        be True.
         """
         if hookenv.config()["bundle-site"] == "":
```

Comment:
Picky observation: I'd go with merging both `if` and directly returning the value.

Reply:
Yeh there is a lot of clean up potential, this part of the code was only touched by black in this case. I would suggest changes like this one are out of scope for this PR

Reply:
Works for me